util/rom-ext-manifest-generator.py

```diff
@@ -0,0 +1,106 @@
+#!/usr/bin/env python3
+# Copyright lowRISC contributors.
+# Licensed under the Apache License, Version 2.0, see LICENSE for details.
+# SPDX-License-Identifier: Apache-2.0
+
+import argparse
+import copy
+from pathlib import Path
+
+import hjson
+from mako.template import Template
+
+from topgen.c import MemoryRegion, Name
+
+DESC = """TODO rom-ext-manifest-gen"""
+
+USAGE = '''
+  rom-ext-manifest-generator [options]
+  rom-ext-manifest-generator [options] <input>
+  rom-ext-manifest-generator (-h | --help)
+'''
+
+# This file is $REPO_TOP/util/rom-ext-manifest-generator.py, so it takes two
+# parent() calls to get back to the top.
+REPO_TOP = Path(__file__).resolve().parent.parent
+
+
+# Adds `size_in_bytes` entry to every parsed ROM_EXT manifest field.
+# fields[N]    =>    fields[N]
+#   - ...              - ...
+#   - size             - size
+#                      - size_in_bytes
+def add_size_in_bytes(fields):
+    for field in fields:
+        field['size_in_bytes'] = str(int(field['size'], 0) // 8)
```

Comment:
Similar to @lenary's comment regarding using strings for integers in the input file, we should also not do it in this file.

Reply:
Done.